test/e2e/storage/persistent_volumes-local.go

```diff
@@ -400,6 +400,25 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 			err = framework.IssueSSHCommand(fileDoesntExistCmd, framework.TestContext.Provider, config.node0)
 			Expect(err).NotTo(HaveOccurred())
 
+			By("Deleting provisioner daemonset")
+			deleteProvisionerDaemonset(config)
```

Comment:
Should there be a `framework.SkipUnlessProviderIs(framework.ProvidersWithSSH...)` check in one of the `BeforeEach`s?
`

Reply:
I think all local PV tests require ssh, so it could go at the very top

Reply:
I've added ProvidersWithSSH at the top.